content/news/2025-08-04-gsoc-2025-report-armaan-chowfin.md

```diff
@@ -123,18 +290,46 @@ While buffer underruns occur due to DAC starvation regardless of the buffer cont
 
 In these scenarios, resampling is a corrective procedure that transforms audio sampled at the input sampling rate to match the DAC’s expected output rate.
 
-Resampling may also be used to *induce* tempo ramping. For a record sampled at 96kHz with a DAC also at 96kHz, scaling tempo by a factor of 3 means we want to pass 3x frames to the DAC on each callback than we would during standard playback.
+##### To induce tempo ramping
```

Comment:
Here a bief explaination is usefull. 

Mixxx acts on a buffer chunks eg. 20 ms. The scratch control command is taken into account between these buffers. To have a steady tempo without clicks and pops, a ramp is applied. If a user changes tempo from 1x to 2x. One buffer is used to slowly change the tempo from 1x to 2x.